### PR TITLE
Integration of IEEE into NFDI-Search-Engine

### PR DESCRIPTION
**Pull request by Najeeb-Shams**
**Description:**
This pull request includes the integration of IEEE (with a sample data record) into the NFDI search engine. Please note that the data will be updated once the IEEE API key is activated.

**Changes Made:**
- Added a new file "search_ieee.py" containing the script for searching IEEE data.
- Synchronized and updated all the relevant parts of the project to support the integration.

Please review and merge this pull request. 
Thank you!

### CODE REVIEW

File: search_ieee.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                            "full_name": "Varsha; P.S. Grover; Laxmi Ahuja"
+                        }
+                    ],
+                    "doi": "https://ieeexplore.ieee.org/document/9596287",
+                    "datepublished": "2021"
+                    
+                }
+            ]
+        }
+        for article in data["articles"]:
```
Comment (huntila) [inline search_ieee.py:37]: This is a static value. The IEEE object is not updated. It always display the same result.

**Review by huntila — CHANGES_REQUESTED** [review]
The searching is not working!
Screenshot 2023-05-31 at 18 52 14

You are trying to directly merge your changes to the develop branch. Please follow the following steps:

- create a new branch from the recently updated develop branch.
- commit the changes to your new branch
- create a pull request for you changes

Besides, the pull request has a conflict.

File: search_ieee.py (hunk 1 above)
Comment (Najeeb-Shams) [reply search_ieee.py:37]: As mentioned in the pull request description, this is a static record only intended as a temporary sample. Once the API key is activated, actual data will replace the static record.

**Comment by Najeeb-Shams** [comment]
@huntila, what should we do with this pull request?

**Pull request closed without merging**